R/build.sh

```diff
@@ -49,7 +47,11 @@ apt-get -qq update &&
     sed -i 's#/build/dest#$(R_ROOT_DIR)#g' /build/dest/lib/R/etc/Makeconf &&
     cp /usr/lib/libgfortran* /build/dest/lib/ &&
     cp /usr/lib/libgomp* /build/dest/lib/ &&
-    cp /usr/lib/libblas.so.3gf /build/dest/lib &&
+    cp /usr/lib/libblas* /build/dest/lib -R &&
```

Comment:
Is the `-R` intentional? If so, it should probably precede the file arguments (although I'm not surprised it works here).